src/Hastlayer/Hast.Xilinx/Drivers/Nexys4DdrDriver.cs

```diff
@@ -4,9 +4,11 @@ namespace Hast.Xilinx.Drivers;
 
 public class Nexys4DdrDriver : NexysDriverBase
 {
-    public const string DeviceName = "Nexys4 DDR";
+    public const string Nexys4Ddr = "Nexys4 DDR";
```

Comment:
Why not `DeviceName`?

Reply:
I created an `abstract string DeviceName { get; }` property in the `DeviceDriverBase` base class so it was clashing.